bug/operation.go

```diff
@@ -49,6 +49,8 @@ type Operation interface {
 	GetMetadata(key string) (string, bool)
 	// AllMetadata return all metadata for this operation
 	AllMetadata() map[string]string
+	// GetAuthor return author identity
+	GetAuthor() identity.Interface
```

Comment:
So many inconsistency in this ... :-| `Get`, no `Get` ...

Reply:
The `baseOp` already have the `Author` field should i rename the method name ?

Reply:
Haaa, that's why I did it that way. Leave it like this.